examples/tutorial/tutorial_02_clock.ml

```diff
@@ -72,11 +72,11 @@ let init () =
   State.add clock1 object1;
   State.add clock2 object2;
 
-  Orx.Clock.register clock1 update Main Normal;
-  Orx.Clock.register clock2 update Main Normal;
+  Orx.Clock.register clock1 update;
```

Comment:
Replaced these with default values since most of the time `Main` and `Normal` are fine.